variables.tf

```diff
@@ -159,6 +159,11 @@ variable "create_certbot_lambda" {
   description = "Boolean to create the certbot lambda to update the letsencrypt cert for the test controller."
   default     = true
 }
+variable "lambda_build_in_docker" {
```

Comment:
This is a different name in case there are additional lambda functions added in the future. That way we can just set a single variable when we want to enable or disable building lambda functions in docker.